### ISSUE TITLE
RHEL 8.0 Repo missing

### ISSUE THREAD

**Issue by hubs032** [microsoft/linux-package-repositories#113]
**Describe the issue**
This URL has no directory for 8.0, but did yesterday, and has a .0 for all other supported versions.

**When did the issue occur?**
First notice 5pm CST on 2/16

**If applicable, what package did you attempt to install, and from which repo?**

**Steps to Reproduce**
Try to download  https://packages.microsoft.com/rhel/8.0/prod/repodata/repomd.xml which is configured by default on Azure RHEL VMs.

**Actual Result**
404

**Expected Result**
200

**Screenshots**
![image](https://github.com/microsoft/linux-package-repositories/assets/79535810/d25213e1-2182-4124-affa-513c674ede40)

**Additional context**

**Comment by Ksurender** [comment]
I have also been running into this. First saw this on February 14th EST.

**Comment by Ksurender** [comment]
This is having a bad interaction with Microsoft Defender for Azure. It will install a prod.repo onto your VM that references the non-existant 8.0 directory. After it does this any yum install commands error out.

**Comment by Ksurender** [comment]
> This is having a bad interaction with Microsoft Defender for Azure. It will install a prod.repo onto your VM that references the non-existant 8.0 directory. After it does this any yum install commands error out.

Here is the corresponding issue in the defender repo:

https://github.com/microsoft/mdatp-xplat/issues/128

**Comment by acheng-01** [comment]
Hi all, thank you for bringing this to our attention. We did have a [prod.repo](https://packages.microsoft.com/config/rhel/8.0/) referencing the 8.0 endpoint but that endpoint had not existed before today, which may have been an oversight on our end - apologies for the inconvenience.

The repos were instead being mapped to rhel/8. We've just now added a new distribution endpoint at rhel/8.0 in addition to rhel/8, which should also resolve that Microsoft Defender issue. I will post a comment there as well, but this 404 issue should be resolved. We will continue monitoring this issue for the next few days in case publishers are still having issues. Thank you for your patience!
